stylesheet.css

```diff
@@ -1,11 +1,75 @@
 html {
     box-sizing: border-box;
+    font-family: 'Source Sans Pro';
+    scroll-behavior: smooth;
 }
 
 * {
     box-sizing: inherit;
+    margin:0px;
+    padding: 0px;
 }
+html {
+    background-color:#484f4f;
+    color: ghostwhite;
+}
+header {
+    display: flex;
+    justify-content: space-between;
+    flex-direction: row;
+    align-items: baseline;
+    background-color: rgba(140, 163, 163, 0.8);
+    position: fixed; 
+    width: 100%; 
+}
+
+nav {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: flex-end;
+    text-decoration: none;
+    text-transform: uppercase;
+}
+
+nav a {
+    margin-left: 20px;
+}
+
+a:link, a:visited{
+    color: inherit;
+    text-decoration: none;
```

Comment:
Links are typically displayed with an underline in the world wide web. If you remove the text-decoration please make sure that people can actually recognize it as links.